### PR TITLE
add vultisigner local setup guide in readme

### PR DESCRIPTION
**Pull request by 0xkobe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,11 +44,21 @@ This component handles the background jobs for key generation and transaction si
...
 
 #### Job: Transaction Signing
 
 - Triggered when a transaction signing request is made.
 - Joins the transaction signing session using the provided session ID.
 - Waits for all parties to join and complete the signing process.
+
+## How to setup vultisigner to run locally?
+
+- Setup redis using docker
```
Comment (johnnyluo) [inline README.md:57]: can you please provided detail , how to start redis using docker, we have docker compose file

File: README.md [hunk 2]
```diff
@@ -43,11 +44,21 @@ This component handles the background jobs for key generation and transaction si
...
+- Open Makefile and confirm relevant values are correct. Update if needed.(REACT_APP_VULTISIG_RELAYER_URL, REACT_APP_VULTISIGNER_USER, REACT_APP_VULTISIGNER_PASSWORD)
+- Register a user into redis. The user name and password should be same with REACT_APP_VULTISIGNER_USER and REACT_APP_VULTISIGNER_PASSWORD
```
Comment (johnnyluo) [inline README.md:59]: provide command etc?

File: docker-compose.yaml [hunk 3]
```diff
@@ -7,13 +7,13 @@ services:
...
 
-  asynqmon:
-    image: hibiken/asynqmon:latest
-    container_name: signer-asynqmon
-    ports:
-      - "${ASYNQMON_PORT:-8020}:8080"
-    environment:
-      REDIS_URL: redis://signer-redis:6379/0
+  # asynqmon:
+  #   image: hibiken/asynqmon:latest
```
Comment (johnnyluo) [inline docker-compose.yaml:11]: why comment out this?

Comment (0xkobe) [reply docker-compose.yaml:11]: It is not needed and that docker image not available now.
Let me remove it, rather than commen out.

**Review by johnnyluo — APPROVED** [review]
LGTM

**Pull request merged**